guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc

```diff
@@ -39,14 +39,22 @@ On the Discovered Hosts page, power off and then delete the discovered hosts.
 From the *Select an Organization* menu, select each organization in turn and repeat the process to power off and delete the discovered hosts.
 Make a note to reboot these hosts when the upgrade is complete.
 ifdef::satellite[]
-. Ensure that the {Project} Maintenance repository is enabled:
+. Ensure that the {Project} Maintenance repository and the {Project} {ProjectVersion} repository are enabled:
 +
 [options="nowrap" subs="attributes"]
 ----
 # subscription-manager repos --enable \
 {RepoRHEL8ServerSatelliteMaintenanceProductVersion}
+
+# subscription-manager repos --enable \
+{RepoRHEL8ServerSatelliteServerProductVersion}
```

Comment:
We try to keep it to one command per instruction/step so I'd merge the two into a single command.

```suggestion
# subscription-manager repos \
--enable {RepoRHEL8ServerSatelliteMaintenanceProductVersion} \
--enable {RepoRHEL8ServerSatelliteServerProductVersion}
```

Reply:
Addressed :+1: